packages/expo-file-system/ios/FileSystemModule.swift

```diff
@@ -256,20 +256,19 @@ public final class FileSystemModule: Module {
 
     AsyncFunction("getFreeDiskStorageAsync") { () -> Int64 in
     // Uses required reason API based on the following reason: E174.1 85F4.1
-      var keyToQuery: URLResourceKey {
 #if !os(tvOS)
-        return .volumeAvailableCapacityForImportantUsageKey
-#else
-        return .volumeAvailableCapacity
-#endif
-      }
-
-      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [keyToQuery])
-
+      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [.volumeAvailableCapacityForImportantUsageKey])
       guard let availableCapacity = resourceValues?.volumeAvailableCapacityForImportantUsage else {
         throw CannotDetermineDiskCapacity()
       }
       return availableCapacity
+#else
+      let resourceValues = try getResourceValues(from: documentDirectory, forKeys: [.volumeAvailableCapacityKey])
```

Comment:
In tvOS, `.documentDirectory` is not available for writing. You should probably be checking `.cachesDirectory` instead.